lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -61,11 +88,22 @@ def on_block(node)
           return unless regular_or_focused_example?(node)
           return if including_any_expectation?(node)
           return if including_any_skip_example?(node)
+          return if including_any_allowed_pattern?(node)
 
           add_offense(node)
         end
 
         alias on_numblock on_block
+
+        private
+
+        def including_any_allowed_pattern?(node)
+          node.each_descendant(:send) do |send_node|
```

Comment:
Wondering if `node.each_descendant(:send).find { |send_node| ... }` would work to avoid returning from a block.

Reply:
Thank you very much. It certainly looks better as you suggested, so I will make the change.